File: vantix-sdk/python/test_sdk.py
# Quick Python SDK verification script
from vantix import VantixClient

def test():
    client = VantixClient(endpoint="http://localhost:5000")
    try:
        att = client.get_attestation()
        print("Vantix Gateway Attestation:", att.get("status", "OK"))
    except Exception as e:
        print("Gateway check:", e)

if __name__ == "__main__":
    test()

File: vantix-sdk/js/test_sdk.js
// Quick SDK verification script
const { VantixClient } = require("./vantix");

async function run() {
  const client = new VantixClient({ endpoint: "http://localhost:5000" });
  try {
    console.log("Checking Vantix attestation...");
    const att = await client.getAttestation();
    console.log("Attestation:", att.status || "OK");
  } catch (err) {
    console.log("Gateway offline or response:", err.message);
  }
}

if (require.main === module) {
  run();
}
